### DISCUSSION TITLE
Should we move workflows to a separate repo?

### DISCUSSION THREAD

**Discussion by SaschaMann** [exercism/problem-specifications#2015] [Polls]
We have an increasing number of workflows in this repo that add utilities (`/format` and `/rebase`) or check for correctness or formatting. As a result, we're also seeing a [fairly large number of Dependabot PRs](https://github.com/exercism/problem-specifications/pulls?q=is%3Apr+author%3Aapp%2Fdependabot). Many of which aren't relevant to this repo.

Given that a significant number of people watch this repo and that merging those PRs takes three reviews, I wonder if it wouldn't make sense to move them to reusable workflows in a designated repo. This would allow us to largely eliminate Dependabot PRs to this repo directly. They could be handled with easier review requirements and without spamming people who aren't interested in them anyway.

The major downside I see is that it adds a layer of indirection which makes it harder to figure out what's actually being run.

I haven't made up my mind yet if this is actually a good idea. What do y'all think?

**Comment by wolf99** [comment]
Is it possible to instead reduce the number of reviews required only for dependabit PRs?
(Am not familiar with this functionality of GitHub)

**Reply by SaschaMann** [reply]
There are settings to bypass pull requests but I'm not aware of a way to change the review requirements based on the user who created the PR. There might be ways to auto-approve it via an action and @exercism-bot or so, but that comes with its own issues.

**Reply by wolf99** [reply]
In that case I agree that moving to separate repo sounds like something worth checking out 🙂
